utils/resultstable.go

```diff
@@ -257,11 +257,26 @@ func sortVulnerabilityOrViolationRows(rows []formats.VulnerabilityOrViolationRow
 	sort.Slice(rows, func(i, j int) bool {
 		if rows[i].SeverityNumValue != rows[j].SeverityNumValue {
 			return rows[i].SeverityNumValue > rows[j].SeverityNumValue
+		} else if rows[i].Applicable != rows[j].Applicable {
+			return sortApplicable(rows[i].Applicable, rows[j].Applicable)
+		} else if rows[i].JfrogResearchInformation.SeverityNumValue != rows[j].JfrogResearchInformation.SeverityNumValue {
+			return rows[i].JfrogResearchInformation.SeverityNumValue > rows[j].JfrogResearchInformation.SeverityNumValue
 		}
-		return len(rows[i].FixedVersions) > 0 && len(rows[j].FixedVersions) > 0
+		return rows[i].IssueId > rows[j].IssueId
 	})
 }
 
+func sortApplicable(applicableFirst string, applicableSecond string) bool {
+	applicableMap := make(map[string]int)
+
+	applicableMap[Applicable.String()] = 4
+	applicableMap[NotApplicable.String()] = 3
+	applicableMap[ApplicabilityUndetermined.String()] = 2
+	applicableMap[NotCovered.String()] = 1
```

Comment:
you are creating this map for every call to the `sortApplicable` function, this should be implemented as const.
It addition, there could be scanning without JAS, so another applicable value could be `NotScanned` that you are missing